cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts

```diff
@@ -16,7 +16,7 @@ describe('EDA decision environment- Create, Edit, Delete', () => {
     cy.hasTitle(/^Decision environments$/);
     cy.clickButton(/^Create decision environment$/);
     cy.typeInputByLabel(/^Name$/, de_name);
-    cy.typeInputByLabel(/^Registry URL$/, 'quay.io/ansible/ansible-rulebook:main');
+    cy.typeInputByLabel(/^Registry URL$/, 'https://quay.io/ansible/ansible-rulebook:main');
```

Comment:
Hi @jamestalton @lgalis I think this is wrong, docker images urls have not http schema. I think the previous value is the correct one.